internal/rbd/nodeserver.go

```diff
@@ -398,8 +398,8 @@ func (ns *NodeServer) stageTransaction(
 	}
 	transaction.devicePath = devicePath
 
-	log.DebugLog(ctx, "rbd image: %s/%s was successfully mapped at %s\n",
-		req.GetVolumeId(), volOptions.Pool, devicePath)
+	log.DebugLog(ctx, "rbd image: %s was successfully mapped at %s\n",
+		volOptions, devicePath)
```

Comment:
@Madhu-1 
if you want to log all volOptions maybe you should use %v?

Reply:
From the commit-msg, I understand you wanted `pool/image` but unfortunately that is not reflected in your chanages.

Reply:
it's only logging the poolname and image name.  rbdImage already have a Sting method implemented at https://github.com/ceph/ceph-csi/blob/devel/internal/rbd/rbd_util.go#L315-L322

Reply:
Sorry for the noise, yep!